framework/src/media/audio/audio_manager.h

```diff
@@ -300,31 +301,45 @@ unsigned int get_output_frames_to_byte(unsigned int frames);
  * Description:
  *   Get the number of frames for the given byte size in output stream.
  *
- * Input parameters:
+ * Input parameter:
  *   bytes: the target of which frame count is returned.
  *
- * Returned Value:
+ * Return Value:
  *   On success, the number of frames in output stream. Otherwise, 0.
  ****************************************************************************/
 unsigned int get_output_bytes_to_frame(unsigned int bytes);
 
 /****************************************************************************
- * Name: get_audio_volume
+ * Name: get_max_audio_gain
  *
  * Description:
- *   Get the maximum volume level of an audio device.
+ *   Get the maximum volume level of the current input audio device.
  *
- * Returned Value: The maximum volume level.
+ * Return Value:
+ *   Unless CONFIG_AUDIO_EXCLUDE_MIC_GAIN is set, the maximum volume level.
+ *   Otherwise, AUDIO_MANAGER_VOLUME_NOT_SUPPORT.
  ****************************************************************************/
-uint16_t get_max_audio_volume(void);
+audio_manager_volume_t get_max_audio_gain(void);
```

Comment:
`audio_manager_volume_t` is defined at audio_manager.c
It makes build fail.

Reply:
Fixed, thx.